public/app/features/explore/utils/links.test.ts

```diff
@@ -28,6 +34,12 @@ describe('explore links utils', () => {
           { type: 'custom', name: 'test', current: { value: 'foo' } },
         ])
       );
+
+      window.open = jest.fn();
```

Comment:
I think we need to restore the original method or use `jest.spyOn` to restore it properly ([more info](https://divotion.com/blog/auto-reset-mocks-restore-spies-in-jest#how-to-properly-spy-on-existing-methods)).